sound/soc/intel/boards/sof_sdw.c

```diff
@@ -42,8 +42,7 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 		},
```

Comment:
Very minor: "from Dell laptop" is a bit odd, is it some specific laptop, all laptops, or? 
The patch seems to remove from all laptops, so mayb ejust
"remove SOF_SDW_NO_AGGREGATION from Dell laptops"

Reply:
> Very minor: "from Dell laptop" is a bit odd, is it some specific laptop, all laptops, or?
> The patch seems to remove from all laptops, so mayb ejust
> "remove SOF_SDW_NO_AGGREGATION from Dell laptops"

Thanks @kv2019i it is for all Dell laptops indeed.